src/types/types.ts

```diff
@@ -2174,6 +2176,11 @@ export type UrlName = string;
  */
 export type ValidDomains = string[];
 
+/**
+ * A set of domains hosting applications used with the project. If missing, a default set of domains will be used. Once application domains are set, they may be individually removed, but cannot be removed entirely in order to fall back to the default list.
```

Comment:
```suggestion
 * A set of domains used within the project for application hosting. If missing, a default set of domains will be used. Once application domains are set, they may be individually removed. However, the removal of all available domains means that login to the project won't be permitted.
```

Reply:
@hhenriques you know it is copied from service contracts docs, right?

Reply:
I adjusted the description to match the service contract exactly.